src/dodal/devices/i24/pmac.py

```diff
@@ -120,12 +122,14 @@ def __init__(
         self,
         pmac_str_sig: SignalRW,
         status_sig: SignalR,
+        counter_sig: SignalR,
         prog_num_sig: SignalRW,
         collection_time_sig: SignalRW,
         name: str = "",
     ) -> None:
         self._signal_ref = Reference(pmac_str_sig)
         self._status_ref = Reference(status_sig)
+        self._counter_ref = Reference(counter_sig)
         self._prog_num_ref = Reference(prog_num_sig)
 
         self._collection_time_ref = Reference(collection_time_sig)
```

Comment:
Should: This collection time is now no longer used, I think we should probably remove it here and where it is set in `mx-bluesky`. Instead setting the timeout for each frame.

Reply:
Moved to https://github.com/DiamondLightSource/mx-bluesky/issues/1050 instead